doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -63,21 +69,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "u = mda.Universe(PSF, DCD)  # closed AdK (PDB ID: 1AKE)"
+    "adk_open = mda.Universe(CRD, DCD2)\n",
+    "adk_closed = mda.Universe(PSF, DCD)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pairwise RMSD"
+    "## Pairwise RMSD of a trajectory to itself"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pairwise RMSDs are an effective way to quickly view similarities and differences in conformations (as measured by RMSD) across a trajectory.\n",
+    "While 1-dimensional RMSD is a quick way to estimate how much a structure changes over time, it can be a misleading measure. It is easy to think that two structures with the same RMSD from the reference frame are also similar; but in fact, they can be very different. Instead, calculating the RMSD of each frame in the trajectory to each other frame in the reference can contain much more information. This measure is often called the pairwise, all-to-all, or 2D RMSD.\n",
+    "\n",
+    "You can calculate a pairwise RMSD of a trajectory to itself, or calculate it to another reference trajectory.\n",
```

Comment:
Considering my question 2 above, maybe it's worth moving this up a little bit so it's more obvious what you mean when you mention "frame in the reference" above instead?

Reply:
Actually with hindsight, maybe the whole of this block should be moved before the "Pairwise RMSD of a trajectory to itself" subheader and instead go into a "Pairwise RMSD" header? Then if possible we could have the "to itself" or "to another reference trajectory" link to the relevant sub-sections?

Reply:
Added a "Background" section to both notebooks now!